package.json

```diff
@@ -10,6 +10,7 @@
     "node": ">=10.0.0"
   },
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```

Comment:
Have you investigated using [shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap)?

Reply:
Good shout- done in 5bbf4aa. I think we still need the npx script though as we want the transitive dependencies upgraded without bumping the direct dependencies.

Reply:
hmm, though I now see Travis chokes on the npx step.